bundles/org.openhab.binding.opensprinkler/src/main/java/org/openhab/binding/opensprinkler/internal/handler/OpenSprinklerBaseBridgeHandler.java

```diff
@@ -40,7 +41,7 @@ public void initialize() {
     /**
      * Threaded scheduled job that periodically syncs the state of the OpenSprinkler device.
```

Comment:
adjust javadoc

Reply:
I've decided to remove the comment and choose a better method name :P